src/Microsoft.ML.Data/Transforms/DropSlotsTransformer.cs

```diff
@@ -14,11 +14,11 @@
 using System.Reflection;
 using System.Text;
 
-[assembly: LoadableClass(DropSlotsTransform.Summary, typeof(DropSlotsTransform), typeof(DropSlotsTransform.Arguments), typeof(SignatureDataTransform),
+[assembly: LoadableClass(DropSlotsTransformer.Summary, typeof(DropSlotsTransformer), typeof(DropSlotsTransformer.Arguments), typeof(SignatureDataTransform),
```

Comment:
>DropSlotsTransformer [](start = 25, length = 20)

Does this match ActionPerformingTransformer?

Reply:
I wouldn't even bother probably considering what @artidoro working on it and it soon become internal/private class.

---
In reply to: [233194607](https://github.com/dotnet/machinelearning/pull/1607#discussion_r233194607) [](ancestors = 233194607)